### PR TITLE
Create orcasound.yaml

### PR DESCRIPTION
**Pull request by prafulfillment**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by awsopendata** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: open-data-registry
* Commit ID: 7595c1d6dbd69e4c1845bdf432091c434484abe4
* Result: FAILED
* [Build Logs](https://onc2c36gc4.execute-api.us-east-1.amazonaws.com/Prod/buildlogs?key=43e414e4-2e0d-4d25-91fe-4dc963c33d3f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jflasher** [comment]
Glad to see this coming back up! Let me know when you're ready for us to review.

**Comment by prafulfillment** [comment]
Updated again @jflasher

File: datasets/orcasound.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    ARN: arn:aws:s3:::archive-orcasound-net
+    Region: us-west-2
+    Type: S3 Bucket
+  - Description: Labeled audio data for ML model development
+    ARN: arn:aws:s3:::acoustic-sandbox
+    Region: us-west-2
+    Type: S3 Bucket
+DataAtWork:
+  Tools & Applications:
+      - Title: Github for our open source projects
```
Comment (jflasher) [inline datasets/orcasound.yaml:38]: I think you've got this indented one level too far, if you pull it back one level, I think this should be good to go!

**Review by jflasher — CHANGES_REQUESTED** [review]

**Review by scottveirs — APPROVED** [review]

**Comment by prafulfillment** [comment]
@jflasher -- should be finalized

**Comment by jflasher** [comment]
Thanks both, now live at https://registry.opendata.aws/orcasound/

**Comment by prafulfillment** [comment]
Amazing thank you for all your help!

**Pull request merged**